src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -269,7 +269,7 @@ public static void ActionMethodExecuting(this ILogger logger, ActionExecutingCon
             }
         }
 
-        public static void ActionMethodExecuted(this ILogger logger, ActionExecutingContext context, IActionResult result)
```

Comment:
Same as the binder - we might invoke the action without an `ActionExecutingContext`